### PR TITLE
Update quickstart.md

### PR DESCRIPTION
**Pull request by fileppb**
## Related issue(s)

Fixes #

### CODE REVIEW

File: quickstart.md [hunk 1]
```diff
@@ -12,13 +12,72 @@ To use components from KCP-Edge you must:
 - [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
 - A Kubernetes cluster (for local testing, consider [kind](http://kind.sigs.k8s.io))
 
+#### Example (Centos)
+
+##### install git, install go version 1.20, set environment variables 
+```
+dnf install git
```
Comment (clubanderson) [inline quickstart.md:19]: should we also install 'wget' for completeness here?

Comment (fileppb) [reply quickstart.md:19]: Good question. Also I don't show installation of docker or curl. I'll add those steps

Comment (MikeSpreitzer) [reply quickstart.md:19]: IMO, for common stuff, it is OK to simply have a list of pre-reqs, not take responsibility for documenting how to install them.

**Comment by clubanderson** [comment]
@fileppb can you move your changes in this PR to the QuickStart doc in the docs site at KCP-edge.ip?  You would need to make the changes to that file instead of the root QuickStart.md. I removed the root QuickStart.md because it was redundant.

**Comment by clubanderson** [comment]
closing this and re-opening to focus on the quickstart doc in the docs.kcp-edge.io site instead.  https://docs.kcp-edge.io/docs/getting-started/quickstart/

**Pull request closed without merging**